var/spack/repos/builtin/packages/gobject-introspection/package.py

```diff
@@ -27,6 +27,9 @@ class GobjectIntrospection(Package):
     depends_on("bison", type="build")
     depends_on("flex", type="build")
     depends_on("pkgconfig", type="build")
+    depends_on('libffi')
+    depends_on('libffi@:3.3', when='@:1.70') # libffi 3.4 was causing seg faults
+    # https://gitlab.gnome.org/GNOME/gobject-introspection/-/merge_requests/283
```

Comment:
Nitpick: Please add the comment above the line it refers to.

Reply:
done